src/mlpack/methods/hmm/hmm_impl.hpp

```diff
@@ -620,6 +630,18 @@ void HMM<Distribution>::Backward(const arma::mat& dataSeq,
   // The last element probability is 1.
   backwardLogProb.col(dataSeq.n_cols - 1).fill(0);
 
+  // Define a variable to store the value of log-probability for dataSeq.
+  arma::mat logProbs(dataSeq.n_cols, transition.n_rows);
+
+  // Save the values of log-probability to logProbs.
+  for (size_t i = 0; i < transition.n_rows; i++)
+  {
+    // Define arma::vec to store log-probability for any dataSeq column.
+    arma::vec logProb;
```

Comment:
Same here.

Reply:
This shouldn't give an error; can you paste the output?